README.md

```diff
@@ -1,20 +1,48 @@
-## Headless mode
+# Salien Bot
 
-Headless mode is a version of this game that is ran without a ui, in your command prompt / terminal
+Salien Bot is a WIP bot for the Salien Minigame that came out for the Steam Summer Sale 2018.
+
+I picked this up because it reminded me of a challenge in programming I had before. I won't be manipulating any state, just injecting mouse clicks and other button presses.
+
+Also, a note: playing this game AT ALL will not net you better rewards! It's only the time you have been on the page. There's no need to waste computer resources. :)
+
+
+## Userscript Mode
+
+Userscript Mode is a version of this bot that is ran as a script on top of your browser, while having a tab with the game open. It is generally easier to use.
+
+Here are the steps to use: 
+
+- Download and install the TamperMonkey or GreaseMonkey addon for your browser of choice - https://tampermonkey.net/
+- Open this link: https://raw.githubusercontent.com/meepen/salien-bot/master/index.user.js with TamperMonkey installed; it should open up a tab that prompts you to install the script
```

Comment:
Can you change this back to https://github.com/meepen/salien-bot/raw/master/index.user.js

Reply:
changed back